cargo-dist/templates/installer/installer.ps1.j2

```diff
@@ -47,6 +50,47 @@ param (
 $app_name = '{{ app_name }}'
 $app_version = '{{ app_version }}'
 
+$install_path = $null
+# If you're reading this install script generated by an app, and
+# this looks strange to you, it's because it's possible for there
+# to be multiple generated paths here. If you're only seeing one
+# $install_path assignment, it's because the app you're looking at
+# has been configured with a single install path.
+{% for install_path in install_paths %}
+if (-Not $install_path) {
+{% if install_path.kind == "CargoHome" %}
+  # This strategy can't fail; always assign it
+  $install_path = @{
+    "strategy" = "CargoHome"
+  }
```

Comment:
I'd still prefer if the powershell impl had the same kind of "no dead code" approach as the shell approach. Especially since you're having to shove a bunch of random keys in here anyway, seems like you're making more logic without unifying anything?